framework-test/src/main/java/org/checkerframework/framework/test/CheckerFrameworkPerDirectoryTest.java

```diff
@@ -137,7 +137,6 @@ protected CheckerFrameworkPerDirectoryTest(
         this.testDir = "tests" + File.separator + testDir;
         this.classpathExtra = classpathExtra;
         this.checkerOptions = new ArrayList<>(Arrays.asList(checkerOptions));
```

Comment:
```suggestion
        this.checkerOptions = Arrays.asList(checkerOptions);
```

Reply:
Can you try whether this works? Now that this doesn't add anything to the list, wrapping in a new ArrayList shouldn't be necessary.